test/mail_dhparams_manual_not_one_dir.bats

```diff
@@ -0,0 +1,53 @@
+load 'test_helper/common'
+
+function setup() {
+    run_setup_file_if_necessary
+}
+
+function teardown() {
+    run_teardown_file_if_necessary
+}
+
+function setup_file() {
+    # copy the custom DHE params in local config
+    cp `pwd`/test/test-files/ssl/ffdhe2048.pem `pwd`/test/config/dhparams.pem
```

Comment:
> Using ffdhe2048 here is a little confusing for me. It's pre-defined and still better than custom generated 2048-bit params right? "user supplied DHE params" might be more appropriate?

You are right, renaming the custom dhe params makes sense. I'll update the comment and rename the file.

> Does that seem better? Suggested improvement:

That's how I did the first test implementation, but the dhparams volume was interfering with `-v "`pwd`/test/config":/tmp/docker-mailserver` generating an empty and  `root` owned and `dhparams.pem` file in `test/config/dhparams.pem`.
Considering that, the copy and remove operation was the easiest way to implement it.

Reply:
> I'll update the comment and rename the file.

Updating the file name isn't required, so long as the comment makes the purpose clear. I was just confused of the intent because of the file name and the comment seemed conflicting.

> Considering that, the copy and remove operation was the easiest way to implement it.

That's fine, I was thinking the suggestion might have been a problem and why you had done it that way.